### ISSUE TITLE
Reorder plugins [idea]

### ISSUE THREAD

**Issue by reteps** [matryer/bitbar#416]
Way to reorder plugins in the top bar?

**Comment by oleander** [comment]
Yes and no. You can move items by pressing the  key while dragging an item right or left in the menu, but they do not persist between reloads.

**Comment by joeynotjoe** [comment]
You can hack this by putting a number in front of the plugin name. They will load in order from lowest to highest (or a-z) from right to left in the menubar.

**Comment by Rhywun** [comment]
> You can hack this by putting a number in front of the plugin name.

I can confirm this works in Sierra. Thanks!

**Comment by mwagstaff** [comment]
Unfortunately, this no longer appears to work (for me, at least) in High Sierra.

**Comment by gingerbeardman** [comment]
See code change in #502
